.github/workflows/import-nixpkgs.yml

```diff
@@ -0,0 +1,63 @@
+name: "Nixpkgs: Hourly import to Elasticsearch"
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 * * * *'
+
+jobs:
+
+  import-nixpkgs:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        channel:
+          - unstable
+          - 21.11
+          - 21.05
+          - 20.09
```

Comment:
We should probably retire `20.09` channel.

Reply:
I was thinking the same since it's EOL

Reply:
Probably belongs in a separate PR though